### ISSUE TITLE
Powershell 10K slows down gnome-terminal

### ISSUE THREAD

**Issue by NavTech270** [romkatv/powerlevel10k#2386]
When running zsh -x to see why zsh was taking a while to start-up, I saw an infinite loop of such commands:

![image](https://github.com/romkatv/powerlevel10k/assets/62112451/da2c2130-19b9-4e64-8d21-e378a4aa13e1)

I'm not sure what any of this means by any help to speed up my zsh would be handy. @romkatv

**Comment by romkatv** [comment]
> Powershell 10K slows down gnome-terminal

What exactly does this mean?

**Comment by NavTech270** [comment]
I ran zsh-bench and it wouldn't output anything; it would just hang. I researched an alternative and I found that running, `zsh -x` shows debug mode of the startup of my zsh. When I ran it, I got the screenshot uploaded above running an infinite amount of times which indicated why my zsh startup was slow. I'm not sure what any of the screenshot means but I assume it's do with the instant prompt feature or something with PowerShell 9k? @romkatv
